sdk/schemaregistry/azure-data-schemaregistry-apacheavro/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.3 (2022-08-10)
 
-### Features Added
-
-### Breaking Changes
+### Other Changes
 
-### Bugs Fixed
+#### Dependency Updates
 
-### Other Changes
+- Update `azure-data-schemaregistry` dependency to `1.2.3`.
```

Comment:
We should mention updates to all our direct dependencies made since the last release.

```suggestion
- Update `azure-data-schemaregistry` dependency to `1.2.3`.
- Update `azure-core` dependency to `1.31.0`.
```

Reply:
Oops. i'll fix this in my next PR. https://github.com/Azure/azure-sdk-for-java/pull/30395